UPDATING.md

```diff
@@ -0,0 +1,8 @@
+# Updating / Migration Guide
+
+This guide lists the changes needed to upgrade from one version of Ably to a newer one when there are breaking changes.
+
+## [Upgrading from v1.2.6]
+
+- `ably.Push.pushEvents` is renamed to `ably.Push.activationEvents`, to be more meaningful. It provides access to events related to setting up push notification, such as activation, deactivation and notification permission. This was done to help future users clearly distinguish between `activationEvents` and `notificationEvents`.
+- When instantiating `Rest` or `Realtime` with an API key, replace `ably.Rest(key: yourApiKey)` with `ably.Rest.fromKey(yourApiKey)`. This was done because using `ably.Rest(key: yourApiKey, options: clientOptions)` was misleading (`yourApiKey` was ignored).
```

Comment:
I think you could soften the tone a little here after 'because' 
Even if it was misleading, would clients be able to pass their key using some other medium(eg. ClientOptions)? If so you could say that "it wasn't neccessary because key was already provided through X. We simplified the API to enable you to use the Rest only using key..."

Reply:
I guess in that line, I am explaining the breaking change and why I **had** to make the change. A breaking change should not just be done to improve it slightly (simplifying the API), but for a very good reason: it was misleading users (if they passed an API key in some cases, it is ignored). So IMHO in this case the tone is important, to inform users that we **had** to make the change, not just for a small improvement.

Reply:
Alternatively, I could say "to fix a bug in API key being ignored". What do you think?